github/dotnet/aspnetcore31/kubernetes/Application/Readme.md

```diff
@@ -0,0 +1,17 @@
+| Language | Framework | Platform | Author |
+| -------- | -------- |--------|--------|
+| ASP.NET Core | .NET Core 3.1 | Azure Web App, Virtual Machines |
+
+
+# ASP.NET Core MVC application
+
+Sample ASP.NET Core MVC application built using Visual Studio 2019.
+
+## License:
+
+See [License](#)
+
+## Contributing
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
```

Comment:
Could you please move the read me file to the root of the repo and get it approved by nikita.